notes_app/client/app/notes/notes_list.component.js

```diff
@@ -15,11 +16,11 @@
         notesService.pagenumToOffset(ctrl.currentPage, ctrl.noteLimit)
       ).then(function(response) {
         ctrl.metadata = response.meta;
-        ctrl.notes = response.objects;
+        ctrl.notes = notesService.data.notes;
```

Comment:
why? this ```update()``` function was made just for this role... to set ```ctrl.metadata``` and ```ctrl.notes``` for rendering right amount of notes for ```currentPage``` relying on ```ctrl.noteLimit``` and calculated offset